data/campaigns/The_Deceivers_Gambit/lua/skill_set.lua

```diff
@@ -48,7 +48,7 @@ local skill_set = {
             id          = "skill_animate_mud",
             label       = label(_"Animate Mud"),
             image       = "icons/animate-mud.png",
-            description = header_passive().._"Learn to recruit <i>Mudcrawlers</i>. Mudcrawlers gain +100% damage and xp\n               while adjacent to you, but dissolve at the end of each scenario.",
+            description = header_passive().._"Learn to recruit <i><ref dst='unit_Mudcrawler'>Mudcrawlers</ref></i>. Mudcrawlers gain +100% damage and xp\n               while adjacent to you, but dissolve at the end of each scenario.",
```

Comment:
I'm not sure if it's worth the effort of you changing, but I've said before that as an abbreviation, 'XP' makes more sense to me than 'xp' in an English-speaking context.